modules/post/multi/gather/enum_vbox.rb

```diff
@@ -29,27 +29,37 @@ def initialize(info={})
 
   def run
     if session.platform =~ /win/
-      res = session.shell_command_token_win32('"c:\Program Files\Oracle\VirtualBox\vboxmanage" list -l vms') || ''
-      if res.include? "The system cannot find the path specified"
-        print_error "VirtualBox does not appear to be installed on this machine"
-        return nil
-      elsif res == "\n"
-        print_status "VirtualBox is installed but this user has no VMs registered. Try another user."
-        return nil
+      if session.type == 'meterpreter'
+        begin
+          res = cmd_exec('c:\\Program Files\\Oracle\\VirtualBox\\vboxmanage', 'list -l vms')
```

Comment:
Should be %programfiles% instead of hard coding the path to C:\. Same on line 45.